control-routing/pom.xml

```diff
@@ -23,8 +23,13 @@
             <artifactId>service-control</artifactId>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
junit 4 has been removed and replaced with junit 5 (jupiter). Update the branch with the latest from java17 branch.